### ISSUE TITLE
Humanoids facing sideways show arms even if they don't have them

### ISSUE THREAD

**Issue by DamianX** [d3athrow/vgstation13#22223]
#### Description of issue
If a spessman is lacking an arm and faces left or right, an arm is still shown. The phantom arm also has a blood overlay for some reason?
Happens with any species.
Possibly linked issue: #9802

![image](https://user-images.githubusercontent.com/6307265/55074911-d83f0700-5091-11e9-93b9-618d687ddbcc.png)

#### Steps to reproduce
- Be spessman with no arms
- Face left or right

#### Server revision
1fcde5a9cdc3610989b799d75f64356e1669a0d2

**Comment by RealestEstate** [comment]
iirc that’s not an arm you’re seeing but the cut out on the jumpsuit sprite, probably because of how the layering is setup

**Comment by DamianX** [comment]
but it looks the same on the naked spessman, no?

**Comment by Exxion** [comment]
There is distinctly no arm there if you zoom in
It's just the cutout on the underwear
We should use layering instead of cutouts but that's a LOT of work at this point

**Comment by DamianX** [comment]
You're right, it's not an arm, it's literally showing the body underneath.
This is probably going to be too much work, as you've said.
